frappe/core/doctype/server_script/server_script.json

```diff
@@ -84,11 +85,19 @@
   {
    "fieldname": "help_html",
    "fieldtype": "HTML"
+  },
+  {
+   "depends_on": "eval:doc.script_type == \"Scheduler Event\"",
+   "fieldname": "event_type",
+   "fieldtype": "Select",
+   "label": "Event Type",
+   "mandatory_depends_on": "eval:doc.script_type == \"Scheduler Event\"",
+   "options": "All\nHourly\nDaily\nWeekly\nMonthly\nYearly\nHourly Long\nDaily Long\nWeekly Long\nMonthly Long"
```

Comment:
Should we rename `All` to `Every minute` ?

Reply:
`All` generally means every 4 minutes with vanilla configuration. This can be changed by setting the `frappe.conf.scheduler_interval` value. `All` isn't a nice label but I don't know what we should change it to, for now 🤔

Reply:
Let's change it to whatever is set in the framework by default.